shared/i18n/info.json

```diff
@@ -61,7 +61,7 @@
       "country_code": "ru",
       "path": "ru_translate_vafin.translation",
 
-      "name": "Russion translation by Vafin, baste on google translate",
+      "name": "Русский (Перевод от 1GTS.RU)",
```

Comment:
Please remove the self advertisement :)
I will may add something like extra notes or description for that, but within in title its inappropriate)

Reply:
> Please remove the self advertisement :)
> I will may add something like extra notes or description for that, but within in title its inappropriate)

How do I change the data already in the open PR? Or need to create a new PR?

Reply:
I removed the link to the site